java/lang/Double/Double.cpp

```diff
@@ -265,13 +262,13 @@ long Double::hashCode(double doubleInput) {
 }
 
 boolean Double::isFinite(double valueDouble) {
-	return (Math::abs(valueDouble) <= MAX_VALUE_DOUBLE);
+    return (Math::abs(valueDouble) <= MAX_VALUE);
 }
 
 boolean Double::isInfinite(double valueDouble) {
-	boolean isPOSITIVE_INFINITY_DOUBLE = (valueDouble == POSITIVE_INFINITY_DOUBLE);
-	boolean isNEGATIVE_INFINITY_DOUBLE = (valueDouble == NEGATIVE_INFINITY_DOUBLE);
-	return( isPOSITIVE_INFINITY_DOUBLE || isNEGATIVE_INFINITY_DOUBLE ) ;
+    boolean isPOSITIVE_INFINITY = (valueDouble == POSITIVE_INFINITY);
```

Comment:
Please rename variable, do not use mixed camel and `_` and upper case

Reply:
Edited. Thanks!